test/examples/positive/llvm/Datatypes.ju

```diff
@@ -3,5 +3,18 @@ mod Datatypes where
 open Prelude
 open LLVM
 
+sig (+) : int -> int -> int
+let (+) = %LLVM.add
+declare infixl (+) 5
+
+type pq : ty = | P int int | Q int int int
+
+sig patternMatch : pq -> int
+let patternMatch (P x y) = x + y
+let patternMatch (Q x y z) = x + y + z
+
 sig main : int
-let main = 0
+let main =
+  -- let pm = patternMatch (P 1 2) in
```

Comment:
why is this commented? do these work?

Reply:
No; that, for example, doesn't work because of #983 .   I shouldn't have commented-out stuff in `positive` golden tests.  Or `negative` ones, for that matter.  Maybe in `to-fix` tests they're tolerable.  I've removed the comments.